.changeset/new-waves-rule.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/core-app-api': patch
+---
+
+- Remove deprecated `@backstage/core-app-api@createApp` which has been replaced by `@backstage/app-defaults#createApp`
```

Comment:
I think typing out the reference probably makes the most sense here, that is "Removed deprecated `createApp` from `@backstage/core-app-api` which ..."